packages/stacked_tools/lib/src/constants/config_constants.dart

```diff
@@ -1,2 +1,2 @@
-/// Defines the path of the stacked configuration file
-const String kConfigFilePath = 'stacked.config.json';
+/// Defines the filename of the stacked configuration file
+const String kConfigFileName = 'stacked.json';
```

Comment:
I like this change, but I think if we're changing it now, and there's people already using it we should make it backwards compatible. 

We don't have to modify a lot of the code to get that working. What we want to do is:
1. Check for the new config file name and perform everything as expected
2. If it's not there, then check for deprecated config file name `stacked.config.json`. When we find this, print a noticeable log that says. "Stacked config file should be renamed from stacked.config.json to stacked.json. Stacked cli will not read stacked.config.json files after the next minor release."
3. Use the stacked.config.json if it's there and continue running the code as normal.